content/tutorials/single-page-tutorials/send-shimmer-tokens-with-javascript.md

```diff
@@ -0,0 +1,265 @@
+# Send Shimmer Tokens with Javascript (client lib)
+
+In this tutorial you will learn how to send Shimmer Tokens in the testnet. We will go through the following steps:
+
+- 1. Get your first address.
+- 2. Get some testnet Tokens.
+- 3. Read your balance.
+- 4. And finally, send the Tokens to another address.
```

Comment:
These should link to their sections, and probably include the first step which is setting up the dev env